clang/test/CodeGenOpenCL/opencl-kernel-call.cl

```diff
@@ -0,0 +1,43 @@
+// RUN: %clang_cc1 -triple amdgcn-unknown-unknown -emit-llvm -o - %s | FileCheck %s
```

Comment:
Might be easier to autogenerate this with `update_cc_test_checks.py`.

Reply:
I think we really need to force auto generate to make (other's) future life much easier.

Reply:
I believe this test makes more sense to have the handpicked patterns to specifically look for the kernel &stub pair generated for the definitions and the call-site replacement with the stub equivalents. In the auto-generated test check sequence it would be quite difficult to examine the real intention of the test. He used the **regex** in the check lines where future changes might occur.
But if everyone thinks having autogenerated check lines are preferable, go for it.

Reply:
the pair can be "detected" or checked by the check lines and any minor change could lead to a test failure, thus makes it more robust.